src/targets/gpu/driver/main.cpp

```diff
@@ -21,6 +21,9 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+
+#define MIGRAPHX_STATIC_DEFINE
```

Comment:
What is this?

Reply:
The driver uses symbols statically defined (linked) and not being imported or exported (dynamic linking). That is how you tell the export.h header file you want static defines. I will remove it from here and move it to the CMake instead of the target property. That would be much cleaner.

Reply:
Actually, these define should not be necessary since we arent compiling static and shared in the same build tree. The export header function from takes care of setting the correct macro automatically.